acceptance_tests/specs/health.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  checkApiHealth,
+  getInternalServiceUrl,
+} from '@datacamp/playwright-helpers';
+import test, { expect } from '@playwright/test';
+
+test('API is healthy', async ({ playwright }) => {
+  const healthy = await checkApiHealth(
+    getInternalServiceUrl('rdocumentaion'),
+    playwright,
+    {
+      healthEndpoint: '/api/local_health',
```

Comment:
do we want to hit the local_health endpoint or just hit the homepage here?

Reply:
this hits the local_health

Reply:
but the other test hits the frontend only